src/Button.js

```diff
@@ -34,7 +34,8 @@ class AriaMenuButtonButton extends React.Component {
       case 'ArrowDown':
         event.preventDefault();
         if (!ambManager.isOpen) {
-          ambManager.openMenu({ focusMenu: true });
+          ambManager.openMenu();
+          ambManager.focusItem(0);
```

Comment:
Shouldn't this be one line `ambManager.openMenu();`? And `openMenu` by default focuses the first item?